scripts/python/packages/parthenon_tools/parthenon_tools/movie2d.py

```diff
@@ -51,6 +52,25 @@
     default=None,
     help="Tensor components of field to plot. Mutally exclusive with --vector-component.",
 )
+parser.add_argument(
+    "--swarm",
+    type=str,
+    default=None,
+    help="Optional particle swarm to overplot figure",
+)
+parser.add_argument(
+    "--swarmcolor",
+    type=str,
+    default=None,
+    help="Optional color of overplotted particle positions. Default is black. You may specify a scalar swarm variable as the color or a matplotlib color string.",
+)
+parser.add_argument(
+    "--maxparticles",
+    metavar="N",
+    type=int,
+    default=100,
+    help="Limit plot to only N particles at most. Default is 100.",
+)
```

Comment:
Not blocking for this PR but `markersize` would be a nice option to have. The default is pretty large for anything more than a very small set of particles. This would set the `s` argument to `scatter()`

Reply:
Done. There's now a `--particlesize` flag, which takes either a float or a string. If it's a float, that's the size in points. If it's a string, the swarm variable with that name is used for the size. (Future improvement: maybe worth a scale factor too)

Reply:
Does the swarm class in `phdf` contain a default marker size that can be updated? I can't see the updated code yet here, maybe it still needs to be pushed

Reply:
Looks like it's present in the branch. 
https://github.com/parthenon-hpc-lab/parthenon/blob/jmm/particle-output/scripts/python/packages/parthenon_tools/parthenon_tools/movie2d.py#L75

The `Swarm` class does not have a default marker size. The default marker size is `matplotlib`s default. If you use a variable then the markersize for each particle is just the float for that variable. Probably it should be rescaled from 0 to some reasonable max (10 points maybe?)

Reply:
Ok now it rescales. I added a `--maxparticlesize` parameter (with the default given by matplotlibs default markersize). So now if you pass in a swarm variable for the `--particlesize` field, you get particles that vary with that variable with the minimum size = 0 and the maximum `maxparticlesize`. Here's the capability for the `id` field for the `particle_tracers` sim:
![0000](https://user-images.githubusercontent.com/1691313/233140052-b3c43faa-c140-441a-8273-c639ac9e7e1b.png)